src/components/timeline/TrackNetwork.js

```diff
@@ -285,7 +285,14 @@ class Network extends PureComponent<Props, State> {
   _onRightClick = () => {
     const { threadIndex, changeRightClickedMarker } = this.props;
     const { hoveredMarkerIndex } = this.state;
-    changeRightClickedMarker(threadIndex, hoveredMarkerIndex);
+    if (hoveredMarkerIndex !== null) {
```

Comment:
I'm not sure why this if check is needed. I think it's also creating an issue. Here's an STR:
1. Open the [deploy preview](https://deploy-preview-3383--perf-html.netlify.app/public/def4zbt1tyg8w0x7nkdtq8g7s6h3ad0ndz7jbzg/network-chart/?globalTrackOrder=de0wc&localTrackOrderByPid=29495-780w6~5123-0~29556-0~526-0~29603-0~29692-01~5023-0~5053-01~29650-0~29726-0~4975-0~29481-0~21926-10&range=5663m2795&thread=e&v=6).
2. Right click on a marker.
3. Right click on an empty space in the timeline before closing the context menu.

When you do this, it still opens the context menu in the area where we right click on the third action. It shouldn't do that, right? For example, when you do this on the marker chart, it closes the context menu and doesn't open it again.

Reply:
Yeah, I believe you're right, thanks for finding the issue :-) I'll do the change.

Reply:
With this change, the menu for vertical indicators don't work anymore... I'll have to find a solution.

Reply:
found a solution!

Reply:
Nice, thanks!